spec/services/ccms/requestors/opponent_organisation_search_requestor_spec.rb

```diff
@@ -47,6 +47,25 @@ module Requestors
         end
       end
 
+      describe "Header" do
+        let(:client_username) { "FakeUser" }
+        let(:client_password) { "FakePassword" }
+        let(:user_role) { "user_role" }
```

Comment:
**nit**, do we need these lets if they are only strings used in the mocks - could just put the string directly in the `and_return` of the mocks

Reply:
otherwise 🎖️

Reply:
👍 removed lets